packages/mail/NodemailerInterface.js

```diff
@@ -0,0 +1,59 @@
+const nodemailer = require('nodemailer')
+
+const { SendMailError } = require('./errors')
+
+const transporter = nodemailer.createTransport(process.env.SEND_MAILS_NODEMAILER_CONNECTION_URL)
```

Comment:
suggestion: only define `transporter` if `SEND_MAILS_NODEMAILER_CONNECTION_URL` is defined and check in `isUsable` if `transporter` is defined